packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/template.success.silver.tsx

```diff
@@ -0,0 +1,329 @@
+import React, { useCallback } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { path } from 'ramda'
+import styled from 'styled-components'
+
+import { Button, Icon, Image, Text } from 'blockchain-info-components'
+import { FlyoutWrapper } from 'components/Flyout'
+import Content from 'components/Flyout/Content'
+import { ModalName } from 'data/modals/types'
+
+import { Props as OwnProps, SuccessStateType } from '.'
+import { TIER_TYPES } from './model'
+import {
+  Disclaimer,
+  RowItemSubTitle,
+  RowItemSubTitleWhite,
+  RowItemTitle,
+  RowItemTitleWhite,
+  RowItemWrapper,
+  StatusCartridge,
+  StatusCartridgeBlue,
+  UpgradeContainer,
+  UpgradeRow,
+  UpgradeRowWithBlueBorder,
+  UpgradeRowWithBorder
+} from './styles'
+
+const Wrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+`
+const Title = styled(Text)`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 7px;
+`
+const IconsContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+`
+const CloseIconContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 32px;
+  height: 32px;
+  border-radius: 50%;
+  background: ${(props) => props.theme.grey000};
+  backdrop-filter: blur(54.3656px);
+  > span {
+    justify-content: center;
+  }
+`
+
+const HeaderWrapper = styled(FlyoutWrapper)`
+  display: flex;
+  flex-direction: column;
+  background-color: ${(props) => props.theme.white};
+`
+
+type Props = OwnProps & SuccessStateType
+
+const Template: React.FC<Props> = (props) => {
+  const startVerification = useCallback(() => {
+    props.modalActions.closeModal(ModalName.TRADING_LIMITS_MODAL)
+    props.modalActions.closeModal(ModalName.UPGRADE_NOW_SILVER_MODAL)
+    props.identityVerificationActions.verifyIdentity({
+      needMoreInfo: false,
+      origin: 'Settings',
+      tier: 2
+    })
+  }, [props.identityVerificationActions, props.modalActions])
+  const { sddEligible, userData, userTiers } = props
+
+  if (!Array.isArray(userTiers)) {
+    return null
+  }
+
+  const userCurrentTier = (path(['tiers', 'current'], userData) as number) ?? 0
+  const sddCheckTier =
+    sddEligible && sddEligible.tier === TIER_TYPES.SILVER_PLUS
+      ? TIER_TYPES.SILVER_PLUS
+      : userCurrentTier
+  const currentTier: number | undefined =
+    userCurrentTier === TIER_TYPES.NONE ? userCurrentTier : sddCheckTier
+  const isUserTierZero = currentTier === TIER_TYPES.NONE
+
+  return (
+    <Wrapper>
+      <HeaderWrapper>
```

Comment:
Not a show-stopper but we really should try to stop re-implementing these flyout headers throughout the app and focus on making the FlyoutHeader in the components directory more flexible for our needs.